src/reducers/dataReducer.jsx

```diff
@@ -8,8 +8,55 @@ export const dataReducer = (state, action) => {
     case "FETCH_PRODUCTS":
       return {
         ...state,
```

Comment:
suggestion: u can store all these action constants separately and use it everywhere required . 
for ex:
 const productActions={
 FETCH_PRODUCTS="FETCH_PRODUCTS"
}